docs/source/cluster_management/sweep/sweep-cli.rst

```diff
@@ -0,0 +1,34 @@
+.. _atlas_sweep_cli:
+
+AtlasDB Sweep Cli
+=================
+
+If you ever need to force a particular table or namespace to be swept immediately, you can run the CLI ``./bin/atlasdb sweep``, which has the following arguments:
+
+.. csv-table::
+   :header: "Short option", "Long option", "Description"
+   :widths: 20, 40, 200
+
+   ``-a``, ``--all``, Sweep all tables.
```

Comment:
Consider adding: "Sweep all tables, in all AtlasDB namespaces."

(Context: it's easy to get confused between AtlasDB namespaces and Cassandra keyspaces)